elastic/integrations
# Add stan package

## What does this PR do?
This PR adds `stan` metricset. 

Still need to add a dashboard for `log` datastream.

## Checklist



## How to test this PR locally
Test the service manually using `elastic-package`:
1.  follow the instructions at https://github.com/elastic/elastic-package/blob/master/docs/howto/connect_service_with_elastic_stack.md#howto-connect-the-service-with-the-elastic-stack start a sample stan service. 
2. Install the stan integration from kibana. Use `/tmp/service_logs/stan.log*` for logs and `elastic-package-service_stan_1:8222` for metrics.
3. Verify that data are flowing for all datastreams and dashboards are working properly.

## Related issues

- Closes https://github.com/elastic/integrations/issues/363


## Screenshots
![Screenshot 2021-01-15 at 12 29 04 PM](https://user-images.githubusercontent.com/11754898/104927096-ac4cee00-59a9-11eb-8a4c-23b29fab22bc.png)
![Screenshot 2021-01-15 at 12 29 33 PM](https://user-images.githubusercontent.com/11754898/104927104-aeaf4800-59a9-11eb-984d-33ccef9c0569.png)
![Screenshot 2021-01-18 at 3 32 46 PM](https://user-images.githubusercontent.com/11754898/104927114-b111a200-59a9-11eb-92f9-41b2998f53ec.png)
![Screenshot 2021-01-15 at 3 02 40 PM](https://user-images.githubusercontent.com/11754898/104927132-b838b000-59a9-11eb-9d20-feb11487278f.png)
![Screenshot 2021-01-15 at 1 38 35 PM](https://user-images.githubusercontent.com/11754898/104927139-ba027380-59a9-11eb-8fb2-70c9b3e8c82b.png)
![Screenshot 2021-01-18 at 3 28 19 PM](https://user-images.githubusercontent.com/11754898/104927167-c1298180-59a9-11eb-9b99-74a2a516fbb1.png)
